File: app/src/main/java/me/pwcong/jpstart/db/JPStartDatabase.java
package me.pwcong.jpstart.db;

import android.content.Context;

import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;

/**
 * Created by Pwcong on 2016/9/28.
 */

public class JPStartDatabase extends SQLiteAssetHelper {

    public static final String DB_NAME = "jpstart.db";
    public static final String DB_TABLE_NAME = "data";
    public static final int DB_VERSON = 1;

    public JPStartDatabase(Context context) {

        super(context, DB_NAME, null, DB_VERSON);

    }

}

File: app/src/main/java/com/github/pwcong/jpstart/db/JPStartDatabase.kt
package com.github.pwcong.jpstart.db

import android.content.Context
import com.readystatesoftware.sqliteasset.SQLiteAssetHelper

class JPStartDatabase(context: Context) :
    SQLiteAssetHelper(context, DB_NAME, null, DB_VERSON) {
    companion object {
        const val DB_NAME: String = "jpstart.db"
        const val DB_TABLE_NAME: String = "data"
        const val DB_VERSON: Int = 1
    }
}